x-pack/plugins/index_management/__jest__/client_integration/index_details_page/index_details_page.test.ts

```diff
@@ -8,22 +8,57 @@
 import { setupEnvironment } from '../helpers';
 import { IndexDetailsPageTestBed, setup } from './index_details_page.helpers';
 import { act } from 'react-dom/test-utils';
-import { httpServiceMock } from '@kbn/core/public/mocks';
 import { IndexDetailsSection } from '../../../public/application/sections/home/index_list/details_page';
+import { testIndexMock } from './mocks';
 
 describe('<IndexDetailsPage />', () => {
   let testBed: IndexDetailsPageTestBed;
   let httpSetup: ReturnType<typeof setupEnvironment>['httpSetup'];
+  let httpRequestsMockHelpers: ReturnType<typeof setupEnvironment>['httpRequestsMockHelpers'];
 
   beforeEach(async () => {
-    httpSetup = httpServiceMock.createSetupContract();
+    const mockEnvironment = setupEnvironment();
+    ({ httpSetup, httpRequestsMockHelpers } = mockEnvironment);
+    // test_index is configured in initialEntries of the memory router
+    httpRequestsMockHelpers.setLoadIndexDetailsResponse('test_index', testIndexMock);
 
     await act(async () => {
-      testBed = await setup(httpSetup);
+      testBed = await setup(httpSetup, {
+        url: {
+          locators: {
+            get: () => ({ navigate: jest.fn() }),
+          },
+        },
+      });
     });
     testBed.component.update();
   });
 
+  describe('error section', () => {
```

Comment:
I'm not testing the loading indicator here, because I think I could re-use what @sabarasaba implemented for enrich policies [here](https://github.com/sabarasaba/kibana/blob/17c97d809b733c4af0387edb16a5ecd08cdae034/x-pack/plugins/index_management/__jest__/client_integration/helpers/http_requests.ts#L179). So I'm planning to add the test later when enrich policies are merged into main.